puppet/modules/jenkins_job_builder/files/config/scripts/gemset_cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ruby=2.0.0
```

Comment:
I've added a proxy one for now. The ruby thing we can handle with something like "localruby=${ruby:=2.0.0}". We could rationalize the way we call rvm which seems different between the two, but possibly not for good reasons. The rake tasks could be conditional'ed in some way....

It seems doable anyway :)